qiskit_nature/second_q/problems/electronic_structure_problem.py

```diff
@@ -69,12 +83,23 @@ def filter_criterion_custom(eigenstate, eigenvalue, aux_values):
             total_angular_momentum_aux = aux_values["AngularMomentum"][0]
 
             return (
-                np.isclose(expected_spin, total_angular_momentum_aux) and
-                np.isclose(expected_num_electrons, num_particles_aux)
+                np.isclose(total_angular_momentum_aux, expected_spin) and
+                np.isclose(num_particles_aux, expected_num_electrons)
             )
 
         solver = NumPyEigensolverFactory(filter_criterion=filter_criterion_spin)
 
+    Attributes:
+        properties: a container for additional observable operator factories.
+        molecule: a container for molecular system data.
+        basis: the electronic basis of all contained orbital coefficients.
+        num_particles: the number of particles (in this case electrons) in the system. When this is
```

Comment:
Is there a reason to have int here rather than require it as a tuple. Elsewhere as in HF and UCC the same named item is of type tuple. If its convenient to take it as an int from some known input that defined elsewhere maybe have a property that always as a getter gives back a tuple to it plugs in elsewhere from the source here without having to check/adapt it

Reply:
I have implemented a variant based on this. This now means that we have `num_particles` handled via a property getter/setter. How do you suggest we document this? Shall I include it in `Attributes:` anyways?
The question can actually be asked for `orbital_occupations`, too

Reply:
I would leave it documented via the property docstring - if you include it in Attributes its going to show up twice in the docs, no.